test/jdk/java/net/Socket/ConnectFailTest.java

```diff
@@ -161,7 +162,8 @@ static List<Socket> sockets() throws Exception {
         Socket socket = new Socket();
         @SuppressWarnings("resource")
         Socket channelSocket = SocketChannel.open().socket();
-        return List.of(socket, channelSocket);
+        Socket proxiedSocket = new Socket(Proxy.NO_PROXY);
```

Comment:
Adding a Socket created with NO_PROXY is a good idea. On naming "proxiedSocket" suggests it's proxies, I think you want noProxySocket :-)